pallets/parachain-staking/src/lib.rs

```diff
@@ -988,11 +988,12 @@ pub mod pallet {
 			bond: BalanceOf<T>,
 			candidate_count: u32,
 		) -> DispatchResultWithPostInfo {
-			let acc = ensure_signed(origin)?;
+			let acc = ensure_signed(origin.clone())?;
+			let is_governance_origin = T::MonetaryGovernanceOrigin::ensure_origin(origin).is_ok();
 			ensure!(!Self::is_candidate(&acc), Error::<T>::CandidateExists);
 			ensure!(!Self::is_delegator(&acc), Error::<T>::DelegatorExists);
 			ensure!(
-				bond >= T::MinCandidateStk::get(),
+				is_governance_origin || bond >= T::MinCandidateStk::get(),
```

Comment:
I didn't think of that solution. I don't think we ever had the concept of an extrinsic that works differently based on the Origin. (I'm not sure it is a good paradigm)

I think that could work in our case, but I think it is better to use a StorageItem to handle the "community collators".

What do you think @librelois ?

Reply:
Yes, it work but it's not intuitive. Instead of a new storage item, I'd rather recommend to create a new call `force_join_candidates` that require a governance origin and skip the bound check

Reply:
Can you check it again, please? @librelois @crystalin